pyaedt/maxwell.py

```diff
@@ -214,7 +214,7 @@ def assign_current(self, object_list, amplitude=1, phase="0deg", solid=True, swa
         object_list : list
             List of objects to assign the current source to.
         amplitude : float, optional
-            Voltage amplitude in mV. The default is ``1``.
+            Voltage amplitude in mA. The default is ``1``.
```

Comment:
voltage in amperes?

Reply:
@beliaev-maksim Correct. it comes from assign_voltage method. I'll fix it